src/FakeItEasy.netstd/project.json

```diff
@@ -2,21 +2,11 @@
   "dependencies": {
     "Castle.Core": "4.0.0-beta001",
     "Microsoft.Extensions.DependencyModel": "1.0.0",
+    "NETStandard.Library":"1.6.0",

```

Comment:
Possibly. I was going to look at trimming, but first wanted to quick get on the standard versions to see if we could at least removed the version mismatch.